framework/db/config_manager.py

```diff
@@ -29,6 +30,18 @@ def IsConvertable(self, value, conv):
 
     def LoadConfigDBFromFile(self, file_path):
         # TODO: Implementy user override mechanism
+        path = os.path.join(os.path.expanduser("~"), os.path.join('.owtf', 'configuration'))
```

Comment:
Add a variable for the configuration files with value `~/.owtf/configuration` to avoid path joins